frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -1015,7 +1015,7 @@ function ReaderHighlight:onShowHighlightDialog(page, index, is_auto_text)
         }
     }
 
-    if self.ui.rolling then
+    if self.ui.rolling and not self.view.highlight.saved[page][index].edited then
         local start_prev = "◁▒▒"
```

Comment:
What does this prevent? Can extend "edited text" bookmarks?
I can understand that: if the text is edited, better not have it replaced with the new text if extended, just wondering about the UX. (No `<< < > >>` without explanations why?)

Reply:
Yes, changing hl boundaries would silently kill the user's text edition, it is not good.
Showing some warning is overengineering here, I believe.
So, when a user meets the lack of `<< < > >>` he recalls his text edition, I hope.

Reply:
I would like to thank you for your efforts, but I can't understand what exactly the question is. What is  `<< < > >>` and where will it be shown?
To be more clear: the main reason I asked for this feature is to correct a bookmark text that comes from a not well recognised book text.
Sorry I don't understand all this technical matters.

Reply:
Now I understand what the matter is!
I think that this possibility (changing the highlight boundaries) should be disabled, since the edited bookmark text is related to a specific text selection.
Perhaps the last option ("do you want to reset the text to be allowed to extend it?") would be more convenient. Or the first one (if the option isn't disabled).

Reply:
I doubt if we should extend this feature out of the Bookmark details window.
Maybe just disable the arrows. It's not mysterious, a user should remember that he made some actions to the highlight.